cluster/ci/config/prow/openshift/jenkins-config-updater.yaml

```diff
@@ -0,0 +1,75 @@
+# Needs the hmac-token secret for decrypting webhook payloads.
+# Needs the oauth-token secret for creating PRs.
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: jenkins-config-updater-prow
+  annotations:
+    "openshift.io/display-name": jenkins-config-updater-prow
+  description: |
+    jenkins-config-updater manifests for prow
+parameters: []
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: jenkins-config-updater-prow
+  spec:
+    selector:
+      app: prow
+      component: jenkins-config-updater-prow
+    ports:
+    - port: 8888
+    type: ClusterIP
+- apiVersion: apps.openshift.io/v1
+  kind: DeploymentConfig
+  metadata:
+    name: jenkins-config-updater-prow
+    labels:
+      app: prow
+      component: jenkins-config-updater-prow
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 2
+    template:
+      metadata:
+        labels:
+          app: prow
+          component: jenkins-config-updater-prow
+      spec:
```

Comment:
Set `deploymentconfig.spec.template.spec.serviceAccountName` to be `jenkins-config-updater`